src/js/components/pages/adjustable-grid.jsx

```diff
@@ -67,6 +67,8 @@ var RespGrid = React.createClass({
 
         ViewLayoutActions.layoutChange(mergedList);
     },
+
+    // APEP TODO review the isLayout property, seems like it is unused in every component but one...
```

Comment:
Properties passed from the grid to components in general need to be reviewed, in some cases we might get updates through both the stores and props.